test/components/formcomponent.test.js

```diff
@@ -70,6 +71,16 @@ lab.experiment('FormComponent', () => {
         expect(viewModel.label).to.exist()
         expect(viewModel.label.text).to.equal('Title')
       })
+      lab.test('label is not page heading', ({ context }) => {
+        const viewModel = context.formComponent.getViewModel({}, formData)
+        expect(viewModel.label).to.exist()
+        expect(viewModel.label.isPageHeading).to.equal(false)
+      })
+      lab.test('label is small', ({ context }) => {
+        const viewModel = context.formComponent.getViewModel({}, formData)
+        expect(viewModel.label).to.exist()
+        expect(viewModel.label.classes).to.equal('govuk-label--s')
+      })
```

Comment:
This functionality already exists and the tests are in the `page.test.js` test suite.